sdk/storage/azure-storage-blob/src/main/java/com/azure/storage/blob/models/PageRangeItem.java

```diff
@@ -0,0 +1,44 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+package com.azure.storage.blob.models;
+
+import com.azure.core.http.HttpRange;
+import com.azure.storage.blob.specialized.PageBlobClient;
+
+/**
+ * Represents a page range on a page blob returned by {@link PageBlobClient#listPageRanges(BlobRange)}.
+ */
+public final class PageRangeItem {
+    private final HttpRange range;
+    private final boolean isClear;
+
+    /**
+     * Creates a new {@code PageRangeItem}.
+     *
+     * @param range The {@link HttpRange}.
+     * @param isClear Whether the bytes are cleared.
+     */
+    public PageRangeItem(HttpRange range, boolean isClear) {
```

Comment:
nit: should the property be named `clear` and have `isClear` accessor?

Reply:
I am open to either if we have a guideline surrounding that.

Reply:
For this, either of these names is okay as this is not going to be directly used by the customer. The getter method should be named `isClear()` regardless of what the arg name used here.